sdcm/nemesis.py

```diff
@@ -1268,3 +1288,8 @@ class NodeRestartWithResharding(Nemesis):
     @log_time_elapsed_and_status
     def disrupt(self):
         self.disrupt_restart_with_resharding()
+
+class AlterEncryption(Nemesis):
```

Comment:
Change also the name of the nemesis class.

Reply:
Sure